include/sys/abd.h

```diff
@@ -38,6 +38,47 @@ extern "C" {
 struct abd; /* forward declaration */
```

Comment:
Do we still need this forward declaration now that we have moved the definition back into abd.h?

Reply:
I don't think so; will fix.